### PR TITLE
Update demo .sh scripts

### PR DESCRIPTION
**Pull request by dpigasin**
**Tickets**: 
- 32201 (.sh only)
- 22208
- 13838 (.sh only)

### CODE REVIEW

File: scripts/demo/demo_security_barrier_camera.sh [hunk 1]
```diff
@@ -76,24 +71,9 @@ if [[ $DISTRO == "centos" ]]; then
         python_binary=python3.6
         pip_binary=pip3.6
```
Comment (generalova-kate) [inline scripts/demo/demo_security_barrier_camera.sh:72]: lets use pip from python -m pip and remove redundant pip_binary

File: scripts/demo/demo_security_barrier_camera.sh [hunk 2]
```diff
@@ -65,29 +65,22 @@ if [[ $DISTRO == "centos" ]]; then
...
         python_binary=python3.5
-        pip_binary=pip3.5
     fi
     if command -v python3.6 >/dev/null 2>&1; then
         python_binary=python3.6
-        pip_binary=pip3.6
     fi
 elif [[ $DISTRO == "ubuntu" ]]; then
     python_binary=python3
-    pip_binary=pip3
```
Comment (generalova-kate) [inline scripts/demo/demo_security_barrier_camera.sh:89]: add check on python 3.8

File: scripts/demo/demo_squeezenet_download_convert_run.bat [hunk 3]
```diff
@@ -205,7 +205,7 @@ if "!MSBUILD_BIN!" == "" (
     GOTO errorHandling
 )
 
-set "SOLUTION_DIR64=%BUILD_FOLDER%\inference_engine_samples_build"
+set "SOLUTION_DIR64=%BUILD_FOLDER%\inference_engine_cpp_samples_build"
```
Comment (generalova-kate) [inline scripts/demo/demo_squeezenet_download_convert_run.bat:208]: please update it for windows bat files as well or as separate PR

File: scripts/demo/demo_security_barrier_camera.sh [hunk 4]
```diff
@@ -11,6 +13,7 @@ usage() {
     echo "Security barrier camera demo that showcases three models coming with the product"
     echo "-d name     specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD are acceptable. Sample will look for a suitable plugin for device specified"
     echo "-help            print help message"
+    echo "-sample-options   specify command line arguments for the sample"
```
Comment (generalova-kate) [inline scripts/demo/demo_security_barrier_camera.sh:16]: the same for bat scrips

File: scripts/demo/demo_security_barrier_camera.sh [hunk 5]
```diff
@@ -116,11 +91,8 @@ if ! command -v $python_binary &>/dev/null; then
     exit 1
 fi
 
-if [[ $DISTRO == "macos" ]]; then
-    "$pip_binary" install -r "$ROOT_DIR/../open_model_zoo/tools/downloader/requirements.in"
-else
-    sudo -E "$pip_binary" install -r "$ROOT_DIR/../open_model_zoo/tools/downloader/requirements.in"
-fi
+"$python_binary" -m pip install -U pip
+"$python_binary" -m pip install -r "$ROOT_DIR/../open_model_zoo/tools/downloader/requirements.in"
```
Comment (generalova-kate) [inline scripts/demo/demo_security_barrier_camera.sh:95]: use --user mode like on Windows.

Comment (generalova-kate) [reply scripts/demo/demo_security_barrier_camera.sh:95]: I have a great idea lets use one venv for all pip installation. Create it in the demo folder.

**Comment by generalova-kate** [comment]
lets use python -m pip install --user -r model-optimizer/requirements.txt
instead of  ./install_prerequisites.sh

**Comment by ilya-lavrenov** [comment]
Please, resolve merge conflicts.

File: scripts/demo/demo_benchmark_app.sh [hunk 6]
```diff
@@ -3,14 +3,20 @@
...
 . "$ROOT_DIR/utils.sh"
 
 usage() {
     echo "Benchmark demo using public SqueezeNet topology"
-    echo "-d name     specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD are acceptable. Sample will look for a suitable plugin for device specified"
-    echo "-help            print help message"
+    echo
+    echo "Options:"
+    echo "  -help                     Print help message"
+    echo "  -d DEVICE                 Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD are acceptable. Sample will look for a suitable plugin for device specified"
```
Comment (generalova-kate) [inline scripts/demo/demo_benchmark_app.sh:17]: remove FPGA

**Review by generalova-kate — APPROVED** [review]

**Review by azhogov — APPROVED** [review]

**Review by generalova-kate — APPROVED** [review]

**Pull request merged**